.github/workflows/add-to-bl.yml

```
@@ -0,0 +1,17 @@
+name: Add new issues to the backlog
+
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    name: Add issue to project
+    runs-on: ubuntu-latest
```

We have been fixing it to a specific version. I don't have a strong opinion on this (in cg we have ubuntu-latest for example), but we should homogenise it here

```suggestion
    runs-on: ubuntu-20.04
```

---

Good suggestions